src/components/admin/index.js

```diff
@@ -1,20 +1,28 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 import { Route, Redirect } from 'react-router-dom';
+import { useLocation } from 'react-router';
 import Header from '../Header';
 import AdminHeader from './AdminHeader';
 import Conseillers from './Conseillers';
+import Territoires from './Territoires';
+import Statistics from '../connected/statistics/Statistics';
 
 function Admin() {
+  const location = useLocation();
 
   const user = useSelector(state => state.authentication.user.user);
   return (
     <>
       <Header linkAccount={user?.name}/>
       <div className="admin">
-        <AdminHeader />
+        { !location.pathname.startsWith('/statistiques') &&
+          <AdminHeader />
+        }
         <Route path={`/accueil`} component={Conseillers} />
-        <Redirect from="/" to="/accueil" />
```

Comment:
@MorganP-projects Pk cette suppression pourr `Route exact path="/" render={() => (<Redirect to="/accueil" />)} />`

Reply:
parce que ça bloquait toutes les routes avec "/"

Reply:
humm ok

Reply:
même en le déplacement à la fin ?

Reply:
il me semble qu'il y était oui